evmrpc/config.go

```diff
@@ -70,6 +70,9 @@ type Config struct {
 
 	// controls whether to have txns go through one by one
 	Slow bool `mapstructure:"slow"`
+
+	// Deny list defines list of methods that EVM RPC should fail fast
+	DenyList []string `mapstructure:"deny_list"`
```

Comment:
would this be comma separated?

Reply:
It will be an array of strings, like ["eth_a", "eth_b"]